cmd/ks-apiserver/app/options/options.go

```diff
@@ -167,6 +165,10 @@ func (s *ServerRunOptions) NewAPIServer(stopCh <-chan struct{}) (*apiserver.APIS
 			}
 			apiServer.CacheClient = cacheClient
 		}
+	} else {
+		klog.Warning("kubesphere-apiserver starts without redis option, it will use in memory cache. The cache data" +
```

Comment:
```suggestion
		klog.Warning("ks-apiserver starts without Redis provided, it will use in-memory cache. This may cause inconsistencies when running ks-apiserver with multiple replicas." )
```